WalletWasabi.Fluent/ViewModels/Wallets/Send/SendSuccessViewModel.cs

```diff
@@ -5,22 +5,27 @@
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets.Send;
 
-[NavigationMetaData(Title = "Payment successful")]
 public partial class SendSuccessViewModel : RoutableViewModel
 {
 	private readonly Wallet _wallet;
 	private readonly SmartTransaction _finalTransaction;
 
-	private SendSuccessViewModel(Wallet wallet, SmartTransaction finalTransaction)
+	private SendSuccessViewModel(Wallet wallet, SmartTransaction finalTransaction, string? title = null, string? caption = null)
 	{
 		_wallet = wallet;
 		_finalTransaction = finalTransaction;
+		Title = title ?? "Payment successful";
+
+		Caption = caption ?? "Your transaction has been successfully sent.";
```

Comment:
The user doesn't "send" a transaction, they send bitcoin, so I guess it makes sense to be precise here.

```suggestion
		Caption = caption ?? "Your bitcoin has been successfully sent.";
```

Reply:
cACK, implementation NACK, the new sentence doesn't feel right.

Reply:
NACK